pages/tutorials/3.0/graphics/shader.md

```diff
@@ -76,17 +76,21 @@ Shaders can also be loaded directly from strings, with the `loadFromMemory` func
 This can be useful if you want to embed the shader source directly into your program.
 
 ```cpp
-constexpr std::string_view vertexShader = \
-    "void main()" \
-    "{" \
-    "    ..." \
-    "}";
-
-constexpr std::string_view fragmentShader = \
-    "void main()" \
-    "{" \
-    "    ..." \
-    "}";
+constexpr std::string_view vertexShader = 
+R"(
+void main()
+{
+    ... 
+}
+)";
+
+constexpr std::string_view fragmentShader = 
+R"(
+void main()
+{
+    ... 
+}
+)";
```

Comment:
```suggestion
constexpr std::string_view vertexShader = R"(
void main()
{
    ... 
}
)";

constexpr std::string_view fragmentShader = R"(
void main()
{
    ... 
}
)";
```
Minor formatting tweak. Otherwise LGTM.